2023/11/30 15:16:34 failed to dial tcp connection, retrying
2023/11/30 15:16:35 failed to dial tcp connection, retrying
[2023-11-30T15:16:35.880Z] info  code-server 4.11.0 85e083580dec27ef19827ff42d3c9257d56ea7e3
[2023-11-30T15:16:35.881Z] info  Using user-data-dir ~/.local/share/code-server
[2023-11-30T15:16:35.895Z] info  Using config file ~/.config/code-server/config.yaml
[2023-11-30T15:16:35.895Z] info  HTTP server listening on http://127.0.0.1:8080/
[2023-11-30T15:16:35.895Z] info    - Authentication is disabled
[2023-11-30T15:16:35.895Z] info    - Not serving HTTPS
====================================>token: 9da02a0c-7410-4b56-9810-e0e9006e31e2
[15:16:38] 




[15:16:38] Extension host agent started.
[15:16:40] [127.0.0.1][8ea2f078][ManagementConnection] New connection established.
[15:16:41] [127.0.0.1][45d6a6f9][ExtensionHostConnection] New connection established.
[15:16:41] [127.0.0.1][45d6a6f9][ExtensionHostConnection] <379> Launched Extension Host Process.
[15:16:45] Using the in-memory credential store as the operating system's credential store could not be accessed. Please see https://aka.ms/vscode-server-keyring on how to set this up. Details: libsecret-1.so.0: cannot open shared object file: No such file or directory
[15:16:55] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /opt/conda/lib/python3.8)
[2023-11-30T15:18:52.344Z] error Failed to get latest version [38;2;140;140;140m{"error":"connect ETIMEDOUT 20.205.243.168:443"}[0m
[15:36:02] [127.0.0.1][8ea2f078][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[15:36:02] [127.0.0.1][45d6a6f9][ExtensionHostConnection] <379> Extension Host Process exited with code: 0, signal: null.
2023/11/30 15:46:03 exit program after timeout
